src/test/java/selenium/webdriver/AlertsTest.java

```diff
@@ -0,0 +1,63 @@
+package selenium.webdriver;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AlertsTest {
+    private WebDriver driver;
+    @BeforeEach
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--ignore-ssl-errors=yes", "--ignore-certificate-errors");
+        driver = new ChromeDriver(options);
+    }
+
+    @Test
+    @DisplayName("JavaScript Confirm Box Test")
+    public void alertBox() throws InterruptedException {
+        driver.get("https://demo.seleniumeasy.com/javascript-alert-box-demo.html");
+        driver.findElement(By.xpath(".//button[@onclick='myAlertFunction()']")).click();
+        Alert alert = new WebDriverWait(driver, Duration.ofSeconds(3)).until(ExpectedConditions.alertIsPresent());
+        alert.accept();
+    }
```

Comment:
Test without assertions is not a test. Each test must be finished by some assertion

Reply:
Done